examples/source_examples/talon_one/README.md

```diff
@@ -0,0 +1,95 @@
+# Talon.one Connector
+
+## Connector overview
+
+This connector fetches event data from Talon.one using the Management API. The connector retrieves application events with pagination support and incremental sync capabilities.
+
+The connector maintains one table:
+- `event`: Contains application event data from Talon.one
+
+## Requirements
+
+* [Supported Python versions](https://github.com/fivetran/fivetran_connector_sdk/blob/main/README.md#requirements)   
+* Operating System:  
+  * Windows 10 or later  
+  * macOS 13 (Ventura) or later
+  * Linux: Distributions such as Ubuntu 20.04 or later, Debian 10 or later, or Amazon Linux 2 or later (arm64 or x86_64)
+
+## Getting started
+
+Refer to the [Setup Guide](https://fivetran.com/docs/connectors/connector-sdk/setup-guide) to get started.
+
+## Features
+
+* Fetches application events from Talon.one Management API
+* Supports incremental sync based on event creation time
+* Implements pagination with configurable page size
+* Rate limiting compliance (3 calls per second)
+* Automatic JSON conversion for list-type fields
+* Checkpointing for reliable data synchronization
+
+## Configuration file
+
+The connector requires configuration with your Talon.one credentials and sync settings:
+
+```json
+{
+    "base_url": "https://<your_subdomain>.us-west1.talon.one/v1/", 
+    "api_key": "<your_api_key>",        
+    "page_size": "1000",                        
+    "initial_sync_start_time": "2025-01-01T00:00:00.00Z"                          
+}
+```
+
+Note: Ensure that the `configuration.json` file is not checked into version control to protect sensitive information.
+
+## Requirements file
+
+The connector uses minimal external dependencies. The `requirements.txt` file should be empty as all required packages are pre-installed in the Fivetran environment.
```

Comment:
```suggestion
The connector uses minimal external dependencies. The `requirements.txt` file should be empty because all required packages are pre-installed in the Fivetran environment.
```